bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Core/SystemUsers/SystemUserService.cs

```diff
@@ -53,21 +53,22 @@ public async Task<List<SystemUser>> GetAllSystemUsersForParty(int id, Cancellati
         return await _systemUserClient.GetSpecificSystemUserReal(partyId, id, cancellationToken);
     }
 
-    public async Task<SystemUser?> CreateSystemUser(int partyId, SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default)
+    public async Task<(SystemUser?, string?)> CreateSystemUser(int partyId, SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default)
```

Comment:
Name tuple arguments - especially when using plain types like `string`.

Reply:
ok, changed to Result type